clientconn_test.go

```diff
@@ -180,91 +180,63 @@ func TestCloseConnectionWhenServerPrefaceNotReceived(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Error while listening. Err: %v", err)
 	}
-	defer server.Close()
+	var (
+		conn1, conn2 net.Conn
+		over         uint32
+	)
+	defer func() {
+		server.Close()
```

Comment:
Can you rename "server" to "lis", please?  It's confusing since it's a listener and not a grpc server.

Reply:
on it.